src/deadline/client/cli/_groups/job_group.py

```diff
@@ -348,28 +347,25 @@ def _download_job_output(
                     new_root = click.prompt(
```

Comment:
I'm not sure how this worked before, but what happens if the user types in a relative path? Ex. `./Downloads`, or something like `~/Desktop` (I'm not sure if Path.resolve() expands `~`, but I imagine we were using `resolve()` before to handle relative paths...)

Reply:
Awesome, thanks for doing the extra testing!